tests/integration/devfile/cmd_devfile_exec_test.go

```diff
@@ -25,25 +26,58 @@ var _ = Describe("odo devfile exec command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("When devfile exec command is executed", func() {
+	When("a component is created", func() {
 
-		It("should execute the given command successfully in the container", func() {
-			utils.ExecCommand(commonVar.Context, cmpName)
-			podName := commonVar.CliRunner.GetRunningPodNameByComponent(cmpName, commonVar.Project)
-			listDir := commonVar.CliRunner.ExecListDir(podName, commonVar.Project, "/projects")
-			Expect(listDir).To(ContainSubstring("blah.js"))
-		})
+		BeforeEach(func() {
+			args := []string{"create", "nodejs", cmpName, "--context", commonVar.Context}
+			helper.Cmd("odo", args...).ShouldPass()
```

Comment:
Is this a new pattern that you're thinking of implementing? If so, I would like here more on it. If not, I would prefer if we simply use `helper.Cmd("odo", "create", "nodejs", cmpName, "--context", commonVar.Context).ShouldPass()` instead, I think it is more readable that way, or at least I have gotten used to reading it like that.

Reply:
No, I was keeping the previous syntax, but I will change with the syntax used anywhere else